scripts/jenkins/ardana/gerrit/build_test_package.py

```diff
@@ -348,13 +348,13 @@ def cleanup_test_packages(self):
                 os.path.join(self.obs_test_project_name, package.name))
 
 
-def test_project_name(change_ids, home_project):
-    return '%s:ardana-ci-%s' % \
-           (home_project, '-'.join(change_ids).replace('/', '-'))
+def test_project_name(change_ids, home_project, build_number):
+    return '%s:ardana-ci-build-%s-changes-%s' % \
```

Comment:
This started out with a simple Gerrit change number, then we added a patch number and now a build number on top of that. Also, if we include more than one Gerrit change (like in a manual job run, which I occasionally use to run custom tests), this project name just gets unnecessarily long.

Now that you're differentiating between projects using the Jenkins build number, I really think we can improve this a little. What would you say to completely removing the Gerrit change numbers from this and just keeping the build number ?

Of course, this will make it more difficult to track down the Gerrit changes for which this project is built, but I have a suggestion for that too: you could add a [description](https://github.com/SUSE-Cloud/automation/blob/master/scripts/jenkins/ardana/gerrit/build_test_package.py#L170) to the build project saying that this and that Gerrit changes are included in this project. You could even make it interesting and include the URLs too.

Reply:
Right, I agree. I had left the change numbers in there as I originally thought just the build number wasn't unique. However, now I can't remember why I thought that and so long as only one job is calling the build script it should be okay.